Remove references to SCA Part 2 (#2909)
theforeman/foreman-documentation: guides/common/modules/proc_creating-a-host.adoc

To use a parameter variable with an Ansible job template, you must add a *Host Parameter*.
ifdef::satellite[]
+
When you create a host, you can set system purpose attributes.

> System purpose attributes help determine which repositories are available on the host as well as with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.

Reviewer: ```suggestion
System purpose attributes help determine which repositories are available on the host.
System purpose attributes also help with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.
```
Does this make sense? I had to re-read the long sentence a few times to parse it :) Splitting it like this or in a similar way would help.

ifdef::satellite[]
+

> When you create a {RHEL} 8 host, you can set system purpose attributes.

Reviewer 2: While we're here, this isn't restricted to RHEL8 either.
